changelog/unreleased/fix-auth-service-jwt.md

```diff
@@ -0,0 +1,5 @@
+Bugfix: Fix the auth service env variable
+
+We the auth service env variable to the service specific name. Before it was configurable via `AUTH_MACHINE_JWT_SECRET` and now is configurable via `AUTH_SERVICE_JWT_SECRET`.
```

Comment:
> We the auth service env variable

@wkloucek you and the `auth-service`, nice :rofl:
Maybe you want to fix that...